rst2pdf/createpdf.py

```diff
@@ -749,6 +751,14 @@ def cleantags(s):
             except OSError:
                 pass
 
+        if self.record_dependencies:
+            with open(self.record_dependencies, 'w') as f:
+                for dep in self.doctree.settings.record_dependencies.list:
+                    f.write(dep)
+                    f.write('\n')
+                f.write(template.filename)
```

Comment:
We only need to write out the `template.filename` if it's a custom cover template and not rst2pdf's default.

Reply:
It shouldn't hurt. It will also trigger a rebuild when rst2pdf is updated or somebody is messing with the default templates.
Compilers (where this feature comes originally from) also include all system includes.